examples/DigitalRain/DigitalRain.ino

```diff
@@ -4,7 +4,6 @@
 #include "Kaleidoscope-LEDControl.h"
 #include "Kaleidoscope-LEDEffect-DigitalRain.h"
 
-#include "LED-Off.h"
```

Comment:
`LEDOff` is specified later in this file. Is this include really not needed?

If `LEDOff` isn't required, on the other hand, leave this removed (and remove one more newline please so there aren't three `\n`s in a row) and remove it below.

If it is required/useful, maybe add a comment saying why you'd want/need the LEDs to switch off?

Reply:
LED-off functionality is now part of the core Kaleidoscope include.  However, to make it active, thus allowing a setting with LED's office, LEDOff must be included in the Plugins. In the name of making a minimal example, I've removed the LEDOff.